src/approve.test.ts

```diff
@@ -16,15 +16,27 @@ test("when a review is successfully created", async () => {
     .post("/repos/hmarr/test/pulls/101/reviews")
     .reply(200, { id: 1 });
 
-  await approve("gh-tok", ghContext());
+  await approve("gh-tok", ghContext(), 0);
```

Comment:
By making the `prNumber` parameter optionally undefined (see below), we can remove all these extra `0` arguments in the tests.

```suggestion
  await approve("gh-tok", ghContext());
```